### PR TITLE
1258 update ltc asset transfer proj std

### PR DESCRIPTION
**Pull request by MeganGeissler**
Please review the updates to meet std requirements. Detailed revisions on Issue. Let me know if my sources are not correct, thanks!

### CODE REVIEW

File: COMPLETE LIST OF SCRIPTS.vbs [hunk 1]
```diff
@@ -2493,7 +2493,7 @@ script_array(script_num).subcategory            = array("HEALTH CARE")
 script_array(script_num).release_date           = #10/01/2000#
 script_array(script_num).hot_topic_link			= ""
 script_array(script_num).used_for_elig			= False
-script_array(script_num).policy_references		= array("")						'SEE Line 58 for format'
+script_array(script_num).policy_references		= array("EPM Asset_Assessment_for_Planning_Purposes http://hcopub.dhs.state.mn.us/epm/2_4_2_1_1.htm", "ONESOURCE Processing_an_Asset_or_Income_Transfer_for_MA-LTC https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126")						'SEE Line 58 for format'
```
Comment (IlseFerris) [inline COMPLETE LIST OF SCRIPTS.vbs:2496]: I think that this EPM reference hits the mark a little closer than the one you found. [2.4.2.1.2 MA-LTC Community Spouse Asset Allowance (state.mn.us)](http://hcopub.dhs.state.mn.us/epm/2_4_2_1_2.htm?rhhlterm=12%20community%20spouse&rhsearch=12%20months%20community%20spouse). 

Also this took me FOREVER to find, and this is a policy that I know.

File: notices/ltc-asset-transfer.vbs [hunk 2]
```diff
@@ -56,40 +57,55 @@ changelog_display
...
+        Text 5, 35, 70, 10, "Resident First Name:"
+        Text 155, 35, 65, 10, "Spouse First Name:"
+        Text 5, 60, 60, 10, "Worker Signature:"
+        GroupBox 15, 80, 100, 30, "LTC Asset Transfer Policies"
+      EndDialog
+
+      Dialog Dialog1
+      cancel_without_confirmation
+
+      If ButtonPressed > 100 Then
```
Comment (IlseFerris) [inline notices/ltc-asset-transfer.vbs:97]: I REALLY like this handling :):)

File: notices/ltc-asset-transfer.vbs [hunk 3]
```diff
@@ -56,40 +57,55 @@ changelog_display
...
+        err_msg = "Loop"
+        If ButtonPressed = onesource_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126"
```
Comment (IlseFerris) [inline notices/ltc-asset-transfer.vbs:99]: Same comment as the Complete List of Scripts (CLoS), I would use this reference: [2.4.2.1.2 MA-LTC Community Spouse Asset Allowance (state.mn.us)](http://hcopub.dhs.state.mn.us/epm/2_4_2_1_2.htm?rhhlterm=12%20community%20spouse&rhsearch=12%20months%20community%20spouse).

File: COMPLETE LIST OF SCRIPTS.vbs (hunk 1 above)
Comment (IlseFerris) [reply COMPLETE LIST OF SCRIPTS.vbs:2496]: Additionally I didn't find any other references to add in POLI TEMP or any where else.

File: notices/ltc-asset-transfer.vbs [hunk 4]
```diff
@@ -56,40 +57,55 @@ changelog_display
...
     Do
-        err_msg = ""
-        Dialog Dialog1
-        cancel_without_confirmation
-        IF IsNumeric(maxis_case_number) = false or len(maxis_case_number) > 8 then err_msg = err_msg & vbNewLine & "* Enter a valid case number."
-        If trim(renewal_footer_month) = "" or len(renewal_footer_month) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal month."
-        If trim(renewal_footer_year) = "" or len(renewal_footer_year) <> 2 then err_msg = err_msg & vbNewLine & "* Enter a two-digit renewal year."
+      err_msg = ""
+
+      Dialog1 = ""
```
Comment (IlseFerris) [inline notices/ltc-asset-transfer.vbs:73]: Good edit box alignment. Can the text be a bit closer to the edit boxes?

File: notices/ltc-asset-transfer.vbs [hunk 5]
```diff
@@ -56,40 +57,55 @@ changelog_display
...
+      Dialog Dialog1
+      cancel_without_confirmation
+
+      If ButtonPressed > 100 Then
+        err_msg = "Loop"
+        If ButtonPressed = onesource_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-170126"
+        If ButtonPressed = epm_URL Then run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe http://hcopub.dhs.state.mn.us/epm/2_4_2_1_1.htm"
+      Else  
+        Call validate_MAXIS_case_number(err_msg, "*")
+        Call validate_footer_month_entry(renewal_footer_month, renewal_footer_year, err_msg, "*")
```
Comment (IlseFerris) [inline notices/ltc-asset-transfer.vbs:103]: The usage on this script is low enough where I don't think we need to auto-fill names, etc. BUT what if there was a 'fill' button next to HC renewal date that when pressed the script would go to STAT/REVW and auto-filled the Health Care ER date?

File: notices/ltc-asset-transfer.vbs [hunk 6]
```diff
@@ -111,55 +127,52 @@ Call start_a_new_spec_memo(memo_opened, True, forms_to_arep, forms_to_swkr, send
...
+Call write_variable_in_CASE_NOTE("Content of the MEMO:")
+Call write_variable_in_CASE_NOTE("The ownership of" & client & "'s assets must be transferred to" & spouse & " to avoid having them counted in future eligibility determinations. You are encouraged to do this as soon as possible. This transfer of assets must be done before" & client & "'s first annual renewal for " & renewal_date & ". Verification of the transfer can be provided at any time.")
+Call write_variable_in_CASE_NOTE("At the first annual renewal in " & renewal_date & ", the value of all assets that list" & client & " as an owner or co-owner will be applied towards the Medical Assistance Asset limit of $3,000.00. If the total value of all countable assets for" & client & " is more than $3,000.00, Medical Assistance may be closed for " & renewal_date & ".")
+Call write_variable_in_CASE_NOTE("---")
+Call write_variable_in_CASE_NOTE(worker_signature)
 
-script_end_procedure_with_error_report("Success! Please review your MEMO and/or CASE:NOTE for accuracy.")
+script_end_procedure_with_error_report("**Review your MEMO and/or CASE:NOTE for accuracy.**")
```
Comment (IlseFerris) [inline notices/ltc-asset-transfer.vbs:151]: There's a newer version of the CLOSING ISSUE DOCUMENTATION. There are a couple of newer elements in there to include.

**Review by IlseFerris — CHANGES_REQUESTED** [review]
OK a few updates, but I'm being real nitpicky for your 1st big solo project. As hard as I could I could not break it, so that's a big win!

File: COMPLETE LIST OF SCRIPTS.vbs (hunk 1 above)
Comment (MeganGeissler) [reply COMPLETE LIST OF SCRIPTS.vbs:2496]: Awesome- I've swapped the link out in both locations.

File: notices/ltc-asset-transfer.vbs [hunk 7]
```diff
@@ -56,49 +57,96 @@ changelog_display
...
+        If ButtonPressed = button_fill Then 
+          Call Check_for_MAXIS(false)                         'Ensuring we are not passworded out  
+          Call validate_MAXIS_case_number(fill_err_msg, "*")
+          If fill_err_msg <> "" Then 
+            MsgBox "Enter the required information specified below before selecting autofill:" & fill_err_msg
+          Else
+            Back_to_SELF
+            Call navigate_to_MAXIS_screen("STAT","REVW")
+            CALL write_value_and_transmit("X", 5, 71)
+            EMReadScreen renewal_footer_month, 2, 9, 27
```
Comment (C-Love) [inline notices/ltc-asset-transfer.vbs:115]: This will autofill '__' for the footer months and years if the case does not have a HC ER coded on REVW.

**Review by C-Love — CHANGES_REQUESTED** [review]
Just one little formatting piece to make sure we don't autofill invalid entries.

**Review by C-Love — APPROVED** [review]
Looks great!

**Pull request merged**